test/whitequark/test_anonymous_blockarg_0.parse-tree-whitequark.exp

```diff
@@ -0,0 +1,5 @@
+s(:def, :foo,
+  s(:args,
+    s(:blockarg, :&)),
```

Comment:
We differ from whitequark/parser implementation by specifying the argument as `&` instead of `nil`.